bot/requirements.txt

```
@@ -0,0 +1 @@
+discord.py==2.3.2
```

```suggestion
discord.py=~2.3.2
```
this will be better if you use automation for updates

---

Same as [this comment](https://github.com/DiscordFi/MasarykReborn/pull/11#discussion_r1539553781).